src/Features/Navigation/Menu.php

```diff
@@ -538,6 +538,7 @@ public function enqueue_data( $menu ) {
 
 		$data = array(
 			'menuItems' => self::get_prepared_menu_item_data(),
+			'postType'  => isset( $_GET['post'] ) ? get_post_type( $_GET['post'] ) : null,
 		);
 
 		$paul = wp_add_inline_script( WC_ADMIN_APP, 'window.wcNavigation = ' . wp_json_encode( $data ), 'before' );
```

Comment:
I know this isn't part of the PR, but I couldn't _not_ ask about `$paul` here.

Reply:
😆 I hadn't even noticed this.  @psealock must have slipped in this easter egg when we transitioned from the nav repo.

Reply:
😆 Wait, you don't name variables after yourself!? I wrote that specifically to avoid it getting committed 😨